src/jit/gentree.cpp

```diff
@@ -17000,7 +17000,13 @@ CORINFO_CLASS_HANDLE Compiler::gtGetStructHandleIfPresent(GenTree* tree)
 #ifdef FEATURE_SIMD
             case GT_SIMD:
                 structHnd = gtGetStructHandleForSIMD(tree->gtType, tree->AsSIMD()->gtSIMDBaseType);
+                break;
 #endif // FEATURE_SIMD
+#if FEATURE_HW_INTRINSICS
+            case GT_HWIntrinsic:
+                structHnd = gtGetStructHandleForHWSIMD(tree->gtType, tree->AsHWIntrinsic()->gtSIMDBaseType);
```

Comment:
BTW, the call site of `gtGetStructHandleForHWSIMD`(SIMD+baseType => class handle) seems not execute. 
Is there any test case that can reach this point? @AndyAyersMS @CarolEidt

Reply:
Seems like you ought to be able to create a test case. Some ideas:
* try / catch with catch type  `Vector128<byte>`
* cast a `Vector128<byte>` to `object`

Reply:
Thank you! I will create and port more tests for `Vector128<T>` and `Vector256<T>` later.